views/email/report.hbs

```diff
@@ -20,43 +20,46 @@
     {{#if breachAlert}}
         {{> breach_alert}}
     {{/if}}
-      {{#if unsafeBreachesForEmail}}
+    {{#if unsafeBreachesForEmail}}
+      <tr>
+        <td colspan="2">
+          <h2 class="body-copy" style="color: #333333; font-size: 22px; font-weight: 300; width: 100%">Website Breaches</h2>
+          {{#if breachAlert}}
+            <p class="body-copy" style="color: #333333; font-size: 15px;">
+              Your email appears in our database of compromised accounts. Read about each one and learn what you can do to protect your passwords. Don't forget to sign up for Firefox Monitor with all the email addresses you use for online accounts.
+            </p>
+          {{/if}}
+        </td>
+      </tr>
+      {{#each unsafeBreachesForEmail.websiteBreaches}}
+        {{> email_single_breach SERVER_URL=../SERVER_URL}}
+      {{/each}}
+      {{#ifCompare unsafeBreachesForEmail.spamLists.length ">" 0}}
```

Comment:
Do we have (or need) some copy explaining what a "spam list" is?

Reply:
Next PR removes the spam lists otherwise we would be needing it.